src/pages/books.js

```diff
@@ -18,12 +18,12 @@ const Books = () => {
   return (
     <>
       <ul>
-        {books.map((book) => (
-          <li key={book.id}>
+        {Object.keys(books).map((id) => (
+          <li key={books.id}>
             <BookItem
-              title={book.bookTitle}
-              author={book.bookAuthor}
-              bookId={book.id}
+              title={books[id][0].title}
+              category={books[id][0].category}
+              bookId={id}
```

Comment:
- [ ] There is a console error about the key property here. It seems that you should pass the `id` to the `key` instead of `books.id` ♻️ 

![image](https://user-images.githubusercontent.com/79876959/145303414-495724ec-d15d-4237-b44f-7d29ce53c2b5.png)